endpoint/get_xxe.php

```diff
@@ -14,6 +14,7 @@
     foreach (glob("$directory/*/*/*.txt") as $filename) {
         $parts = explode("/", $filename);
         list($time, $payload_num, $ip) = explode("-", basename($filename, ".txt"));
+	    // ./xxe_data/<session_id>/<path_id>/<hex_param>/<filename>
```

Comment:
maybe remove this comment or rearrange it to follow the other lines  indentation and a description